Taskfile.yml

```diff
@@ -29,6 +29,10 @@ includes:
     taskfile: ./taskfiles/Taskfile_gotest.yml
     dir: ./taskfiles
 
+  systemtest:
```

Comment:
`task test:system`

https://jitsuin.atlassian.net/wiki/spaces/ENG/pages/3140157463/Common+Tasks+Across+All+Repos

Reply:
Not sure how this will fit in with the existing task runes, which aren't standard either. I'm going to keep as is to minimise code churn. All task files will then need to be standardised again at a later date

Reply:
I'm not sure. I thought the repo was created after we agreed on the standard runes. I can see `task test:unit` and `task test:integration` in here, so `task test:system` should be ok to use?

Reply:
Looks like I misunderstood the structure of the task runes. 

I added a new top level entry called `task test:system`

The original definition still stays. 

```
serhiy@jitsuin-serhiy:~/git/veracity$ task
task: Available tasks for this project:
* build:                 ensure go build works for all modules
* format:                formats the code correctly
* lint:                  lints the go code
* azurite:cleanup:       stop the container and DELETE the data directory identified by AZURITE_DATA_DIR

* azurite:follow-logs:        follow the logs of the azurite container
* azurite:logs:               follow the logs of the azurite container
* azurite:preflight:          stops, cleans and re-starts the emulator providing a clean state
* azurite:start:              start azurite azure local storage emulator in a named docker container
* azurite:stop:               stop azurite azure local storage emulator docker container
* build:clean:                ensure go build works for all modules
* build:fast:                 ensure go build works for all modules
* codequality:all:            run all code quality tasks
* codequality:format:         format sources (go fmt)
* codequality:lint:           Quality assurance of **all** code for desktop
* codequality:modules:        Make all go.mod files consistent
* go:modules:                 tidies the go modules
* gobuild:go:build:           run go build against all go modules
* gotest:go:azurite:          run integration tests which depend only on azurite
* gotest:go:prodpublic:       run the integration tests
* gotest:go:unit:             run unit tests
* systemtest:setup:           Clones the shunit test framework and installs it into the correct folder
* systemtest:test:            Runs the system tests
* test:integration:           run the azurite integration tests
* test:system:                run the system tests
* test:unit:                  run the unit tests
```

Reply:
hmm, that list of definitions is more confusing, having multiple runes for the same thing. Perhaps just put it back how you had it before, for the time being, if it's going to be a big task to bring it neatly in line with the repos.